README.md

```diff
@@ -76,46 +81,82 @@ For example to run all unit tests execute in the build directory:
 ```
 ctest -L unit
 ```
+### Build Instructions for Windows using Visual Studio
+
+First, install Visual Studio and select the "Desktop development with C++" workload. 
```

Comment:
Maybe specify to install visual studio 2017 or later?

Reply:
Updated the readme with 5cebe1c